director/sort.go

```diff
@@ -144,21 +148,41 @@ func checkOverrides(addr net.IP) (coordinate *Coordinate) {
 	return nil
 }
 
-func getLatLong(addr netip.Addr) (lat float64, long float64, err error) {
+func getLatLong(ctx context.Context, addr netip.Addr) (lat float64, long float64, err error) {
 	ip := net.IP(addr.AsSlice())
 	override := checkOverrides(ip)
 	if override != nil {
 		log.Infof("Overriding Geolocation of detected IP (%s) to lat:long %f:%f based on configured overrides", ip.String(), (override.Lat), override.Long)
 		return override.Lat, override.Long, nil
 	}
 
+	network, ok := utils.ApplyIPMask(addr.String())
+	if !ok {
+		log.Warningf("Failed to apply IP mask to address %s", ip.String())
+	}
+
+	project, ok := ctx.Value(ProjectContextKey{}).(string)
+	if !ok {
+		log.Warningf("Failed to get project from context")
+		project = "unknown"
+	}
 	reader := maxMindReader.Load()
 	if reader == nil {
```

Comment:
This should probably be treated differently than the other cases because it indicates "the director can't load a database it should have". If this error is triggered by a client request, I'd be surprised if it isn't triggered by _every_ client request. In that case, it doesn't make a lot of sense to include the network/project information because they don't tell us much. If nothing else, we can be more explicit by setting the source to "director".